content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Package and deploy your models with MLEM
```

Comment:
Saw some talk about changing names of things from package to build today in the channel.  Does this need to be rethought?

Reply:
Not now, it's not decided yet. If it will be, I'll update the page. Thanks!